code/modules/research/techweb/all_nodes.dm

```diff
@@ -369,6 +369,8 @@
 		"integrated_circuit",
 		"module_duplicator",
 		"usb_cable",
+		"comp_list_pick",
+		"assoc_comp_list_pick"
```

Comment:
ABC order

Reply:
also renamed the 2nd var to be inline with the other comps